inc/customizer.php

```diff
@@ -15,6 +15,9 @@ function twentyseventeen_customize_register( $wp_customize ) {
 	$wp_customize->get_setting( 'blogdescription' )->transport  = 'postMessage';
 	$wp_customize->get_setting( 'header_textcolor' )->transport = 'postMessage';
 
+	// Move the Front Page widget area to the Theme Options panel.
+	$wp_customize->get_section( 'sidebar-widgets-front-page' )->panel = 'twentyseventeen_options_panel';
```

Comment:
This is creating an error `Warning: Creating default object from empty value in/twentyseventeen/inc/customizer.php on line 19`